src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CaseDataLiP.java

```diff
@@ -33,6 +34,8 @@ public class CaseDataLiP {
     private AdditionalLipPartyDetails respondent1AdditionalLipPartyDetails;
     @JsonProperty("applicant1AdditionalLipPartyDetails")
     private AdditionalLipPartyDetails applicant1AdditionalLipPartyDetails;
+    @JsonProperty("applicant1RepaymentPlan")
+    private RepaymentPlanLRspec applicant11RepaymentPlan;
```

Comment:
applicant**11**RepaymentPlan - double 1, is that intentional?

Reply:
Good spot - but actually this field (from Neeta's original work on this ticket) isn't used, so I have removed the field.